File: backend/models/BodyMeasurement.cs
using System.ComponentModel.DataAnnotations;

namespace FitTrackBackend.Models;

public class BodyMeasurement
{
    [Key]
    public int MeasurementId { get; set; }

    public int UserId { get; set; }

    public decimal BodyWeight { get; set; }

    public decimal Chest { get; set; }

    public decimal Arms { get; set; }

    public decimal Waist { get; set; }

    public decimal Legs { get; set; }

    public DateTime MeasurementDate { get; set; }
}

File: FitTrackApp/app/src/main/java/com/example/fittrackapp/models/BodyMeasurement.java
package com.example.fittrackapp.models;

public class BodyMeasurement {
    public int measurementId;
    public int userId;
    public double bodyWeight;
    public double chest;
    public double arms;
    public double waist;
    public double legs;
    public String measurementDate;

    public BodyMeasurement(int userId, double bodyWeight, double chest, double arms, double waist, double legs, String measurementDate) {
        this.userId = userId;
        this.bodyWeight = bodyWeight;
        this.chest = chest;
        this.arms = arms;
        this.waist = waist;
        this.legs = legs;
        this.measurementDate = measurementDate;
    }
}